pandas/io/json/_normalize.py

```diff
@@ -94,16 +133,22 @@ def nested_to_record(
             # current dict level  < maximum level provided and
             # only dicts gets recurse-flattened
             # only at level>1 do we rename the rest of the keys
-            if not isinstance(v, dict) or (
-                max_level is not None and level >= max_level
+
+            if (
+                use_key
```

Comment:
don't you need parens here? (use_key and not use_key(k)) ? alternatively you can *always* return a callable lambda key: True